airflow/version.py

```diff
@@ -17,20 +17,5 @@
 # under the License.
 from __future__ import annotations
 
-__all__ = ["version"]
-
-try:
-    import importlib_metadata as metadata
-except ImportError:
-    from importlib import metadata  # type: ignore[no-redef]
-
-try:
-    version = metadata.version("apache-airflow")
-except metadata.PackageNotFoundError:
-    import logging
-
-    log = logging.getLogger(__name__)
-    log.warning("Package metadata could not be found. Overriding it with version found in setup.py")
-    from setup import version
-
-del metadata
+# Compat -- somethings access `airflow.version.version` directly
+from airflow import __version__ as version  # noqa: F401
```

Comment:
Don’t need to `noqa` if you keep the `__all__` line

Reply:
df9faa9ca0